CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## 2.25.1 (Unreleased)
+FEATURES:
+* **New Resource:** sumologic_muting_schedule
```

Comment:
Add (GH-569) to the end of this line - 569 is the pull request

Reply:
updated.